주민_강민주/1주차_미션/CSS핸드북_미션1/HB02.MS1.css

```
@@ -0,0 +1,30 @@
+p {
```

```suggestion
body {
    margin: 0;
    padding: 0;
}
```
이런 식으로 하면 nav바에 padding을 없애서 화면에 꽉차게 구현할 수 있습니다!